server/src/main/java/org/elasticsearch/action/admin/indices/settings/get/GetSettingsRequest.java

```diff
@@ -48,6 +52,16 @@ public GetSettingsRequest indicesOptions(IndicesOptions indicesOptions) {
         return this;
     }
 
+    /**
+     * When include_defaults is set, return default values which are normally suppressed.
+     * This flag is specific to the rest client.
```

Comment:
having moved the default settings logic to the transport action, I think that this is now also applied to the transport client and any request, so not only high-level REST client. Is that correct?

Reply:
I think that this comment needs to be updated.